internal/cli/args_test.go

```diff
@@ -71,6 +72,24 @@ func TestArgsClearAndNewValue(t *testing.T) {
 	require.Contains(t, err.Error(), "--clear cannot be specified with other values")
 }
 
+func TestArgsNoChange(t *testing.T) {
+	f := newServerFixture(t)
+	defer f.TearDown()
+
+	createTiltfile(f, []string{"foo", "bar"})
+
+	cmd := newArgsCmd()
+	out := &bytes.Buffer{}
+	cmd.streams.Out = out
+	cmd.streams.ErrOut = out
```

Comment:
this necessary because the current design... it is the creation of a default IOStream object then setting each property for a test.  It would be much better to not create the defaults and to pass in the IOStream to the cmd IMO.

Reply:
same as previous stream comment